version.json

```diff
@@ -1,5 +1,6 @@
 {
   "version": "3.24",
+  "buildNumberOffset": -1,
```

Comment:
I wouldn't do this for the app. You can use `3.25.0` if that's what you really want for the release and then use `3.26.0-build.{height}` for subsequent builds.

Reply:
OK, guess that's https://github.com/NuGetPackageExplorer/NuGetPackageExplorer/pull/246 ?